### PR TITLE
Upgrade 0503

### PR DESCRIPTION
**Pull request by DonnaWuDongxia**
Verified in Vive, works.
I rebased, so it conflicts.

### CODE REVIEW

**Comment by huningxin** [comment]
Thanks! Could you also rebase this one: https://github.com/huningxin/aframe-city-builder/commit/7734811997059c87f97e50c8b5d4bc72ce16b964? It is about making resource local.

**Comment by DonnaWuDongxia** [comment]
@huningxin updated and verified on Vive.

File: index.html [hunk 1]
```diff
@@ -2,41 +2,56 @@
...
+    <link rel="stylesheet" href="css/bootstrap.min.css" integrity="sha256-916EbMg70RQy9LHiGkXzG8hSg9EdNy97GazNG/aiY1w=" crossorigin="anonymous" />
 
     <!-- components used -->
-    <script src="https://rawgit.com/bryik/aframe-bmfont-text-component/master/dist/aframe-bmfont-text-component.min.js"></script>
-    <script src="https://rawgit.com/ngokevin/kframe/master/components/text/dist/aframe-text-component.min.js"></script>
-    <script src="https://rawgit.com/ngokevin/aframe-animation-component/master/dist/aframe-animation-component.js"></script>
-    <script src="https://rawgit.com/kfarr/aframe-gridhelper/master/dist/aframe-gridhelper-component.min.js"></script>
-    <script src="lib/skyGradient.js"></script>
+    <!-- <script src="https://rawgit.com/fernandojsg/aframe-teleport-controls/master/dist/aframe-teleport-controls.min.js"></script> -->
+    <script src="https://rawgit.com/kfarr/aframe-teleport-controls/master/dist/aframe-teleport-controls.min.js"></script>
```
Comment (huningxin) [inline index.html]: @DonnaWuDongxia , are you able to make following components to be local?

**Comment by huningxin** [comment]
@DonnaWuDongxia , verified on oculus. Could you please make the components to be local? Just like https://github.com/huningxin/aframe-city-builder/commit/7734811997059c87f97e50c8b5d4bc72ce16b964 did. Thanks!

File: index.html (hunk 1 above)
Comment (DonnaWuDongxia) [reply index.html]: Got it. I will update it.

**Comment by DonnaWuDongxia** [comment]
@huningxin updated. Please take a look at it.

**Comment by huningxin** [comment]
verified on oculus. merge.

**Pull request merged**